Jenkinsfile

```diff
@@ -124,7 +124,7 @@ node('vetsgov-general-purpose') {
       if (!commonStages.isDeployable()) { return }
 
       if (commonStages.IS_DEV_BRANCH && commonStages.VAGOV_BUILDTYPES.contains('vagovdev')) {
-        commonStages.runDeploy('deploys/vets-website-vagovdev', ref, false)
```

Comment:
@theodur should we leave both deploys in place for now. I'm thinking about if we need to roll things back it might be nice to not have to redeploy to the old bucket

Reply:
Yeah I think so, I left uploading the full build to the `vagovdev` bucket for easier rollback, so we can keep this too.